Wire.podspec

```diff
@@ -1,11 +1,13 @@
 Pod::Spec.new do |s|
+  version = File.read('./README.md').match(/<version>([0-9.]+)<\/version>/).captures[0]
```

Comment:
Wanna do the same technique, but gradle.properties instead? That'll always be accurate. Sometimes for the readme we skip alpha releases.

Reply:
Yup! And in fact, I was looking for places that were specifying 3.7.0 and missed gradle.properties because it had been bumped to 3.8.0. Will update.